app/src/main/java/com/panhuk/quizapp/MainActivity.kt

```diff
@@ -4,10 +4,12 @@ import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
 import androidx.navigation.NavController
 import androidx.navigation.findNavController
+import com.panhuk.finishfeature.FinishNavigator
 import com.panhuk.menufeature.MenuNavigator
+import com.panhuk.playfeature.PlayNavigator
 import com.panhuk.quizapp.databinding.ActivityMainBinding
 
-class MainActivity : AppCompatActivity(), MenuNavigator {
+class MainActivity : AppCompatActivity(), MenuNavigator, FinishNavigator, PlayNavigator {
```

Comment:
You can aggregate it into 1 interface as
```kotlin
interface ActivityNavigation: MenuNavigator, FinishNavigator, PlayNavigator, ... {}

...

class MainActivity : AppCompatActivity(), ActivityNavigation {
...
```

Reply:
I am not sure because we are using approach where in every feature module we most likely will have an interface and Activity will override it as well